src/GalsimBenchmark.jl

```diff
@@ -94,10 +94,10 @@ end
 function typical_reference_brightness(is_star::Bool)
     source_type_index = is_star ? 1 : 2
     prior_parameters::Model.PriorParams = Model.load_prior()
-    exp(
-        prior_parameters.r_mean[source_type_index]
-        - prior_parameters.r_var[source_type_index]
-    )
+
+    # this is the mode. brightness is log normal.
+    exp(prior_parameters.r_μ[source_type_index]
+            - prior_parameters.r_σ²[source_type_index])
```

Comment:
I changed the name of `r_var` because it's the variance of the `log r`, not of `r`, and it's easy to get them confused.